nodejs/test/unit/Render.spec.js

```diff
@@ -958,3 +980,189 @@ test(`Render - Returns false if font is not known`, () => {
 
 	expect(test).toBe(false);
 });
+
+test(`Render - Render console string in raw mode`, () => {
+	const test = Render(
+		'text',
+		{
+			font: 'console',
+			rawMode: true,
+		},
+		false,
+		1,
+		{ width: 100, height: 10 }
+	);
+
+	expect(test.string).toBe('\r\n\r\ntext\r\n\r\n');
+	expect(test.array).toEqual(['\r\n\r\ntext\r\n\r\n']);
+	expect(test.lines).toBe(1);
+	expect(test.options).toEqual({
+		font: 'console',
+		align: 'left',
+		colors: [],
+		background: 'transparent',
+		letterSpacing: 0,
+		lineHeight: 0,
+		space: true,
+		maxLength: 0,
+		gradient: false,
+		independentGradient: false,
+		transitionGradient: false,
+		env: 'node',
+		rawMode: true,
+	});
+});
+
+test(`Render - Break into new line on smaller viewports in raw mode`, () => {
+	const test = Render('text', { rawMode: true }, false, 1, { width: 20, height: 10 });
+
+	expect(test.string).toBe(
+		'\r\n\r\n' +
+			' ████████╗ ███████╗\r\n' +
+			' ╚══██╔══╝ ██╔════╝\r\n' +
+			'    ██║    █████╗  \r\n' +
+			'    ██║    ██╔══╝  \r\n' +
+			'    ██║    ███████╗\r\n' +
+			'    ╚═╝    ╚══════╝\r\n\r\n' +
+			' ██╗  ██╗ ████████╗\r\n' +
+			' ╚██╗██╔╝ ╚══██╔══╝\r\n' +
+			'  ╚███╔╝     ██║   \r\n' +
+			'  ██╔██╗     ██║   \r\n' +
+			' ██╔╝ ██╗    ██║   \r\n' +
+			' ╚═╝  ╚═╝    ╚═╝   \r\n\r\n'
+	);
+	expect(test.array).toEqual([
+		'\r\n\r\n ████████╗ ███████╗',
```

Comment:
I'd move the `\r\n\r\n` to a new line just like the others so it's more readable.

Reply:
Makes sense. I'll probably have to go over the other tests to see if I can apply this to other places